src/Datadog.Trace/Logging/DatadogSerilogLogger.cs

```diff
@@ -0,0 +1,210 @@
+using System;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using Datadog.Trace.Vendors.Serilog;
+using Datadog.Trace.Vendors.Serilog.Events;
+
+namespace Datadog.Trace.Logging
+{
+    internal class DatadogSerilogLogger : IDatadogLogger
+    {
+        private static readonly object[] NoPropertyValues = new object[0];
+        private readonly ILogger _logger;
+        private readonly ILogRateLimiter _rateLimiter;
+
+        public DatadogSerilogLogger(ILogger logger, ILogRateLimiter rateLimiter)
+        {
+            _logger = logger;
+            _rateLimiter = rateLimiter;
+        }
+
+        public bool IsEnabled(LogEventLevel level) => _logger.IsEnabled(level);
+
+        public void Debug(string messageTemplate, [CallerLineNumber] int sourceLine = 0, [CallerFilePath] string sourceFile = "")
+            => Write(LogEventLevel.Debug, exception: null, messageTemplate, NoPropertyValues, sourceLine, sourceFile);
+
+        public void Debug<T>(string messageTemplate, T property, [CallerLineNumber] int sourceLine = 0, [CallerFilePath] string sourceFile = "")
+            => Write(LogEventLevel.Debug, exception: null, messageTemplate, property, sourceLine, sourceFile);
+
+        public void Debug<T0, T1>(string messageTemplate, T0 property0, T1 property1, [CallerLineNumber] int sourceLine = 0, [CallerFilePath] string sourceFile = "")
```

Comment:
Starting to count at `T0` ? That's weird but ok 😄

Reply:
It's what the underlying `Serilog.Logger`  uses, so though it would be better to be consistent 🙂